### PR TITLE
[node-manager] Add kubernetess version to the NodeGroup CR status field

### PR DESCRIPTION
**Pull request by alex123012**
## Description

This PR adds `status.kubernetesVersion` field to the `NodeGroup`
## Why do we need it, and what problem does it solve?

for Deckhouse UI purposes
## What is the expected result?

kubernetes version will be in status field of `NodeGroup` CR
## Checklist
- [x] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: node-manager
type: chore
summary: Add the `kubernetesVersion` field to the NodeGroup CR status fields.
impact_level: low
```

### CODE REVIEW

File: candi/openapi/doc-ru-node_group.yaml [hunk 1]
```diff
@@ -72,6 +72,8 @@ spec:
                   description: "The summary number of overprovisioned Nodes for this NodeGroup in all zones."
                 error:
                   description: "Error message about possible problems with the group handling."
+                kubernetesVersion:
+                  description: "The current version of kubernetes on the nodes, or the version to which the nodes will be upgraded."
```
Comment (yalosev) [inline candi/openapi/doc-ru-node_group.yaml:76]: doc-ru-* should have RU description :)

File: candi/openapi/node_group.yaml [hunk 2]
```diff
@@ -79,11 +79,12 @@ spec:
                 name:
                   type: string
                   pattern: "^[a-z0-9]([-a-z0-9]*[a-z0-9])?$" # In Amazon cloud, AWSMachineClass name can't contain dots.
-                  maxLength: 42 # One of labels assigned by the cloud instance manager contains full node name,
-                                # the same time, kubernetes labels keys and values can't be longer than 63 characters.
-                                # Length of hashes is 21 characters, so we leave 63-21=42 characters.
-                                # Further, validation webhook will check that length of <cluster prefix>-<group node name>
-                                # fits the 42 characters limitation.
+                  maxLength:
+                    42 # One of labels assigned by the cloud instance manager contains full node name,
```
Comment (yalosev) [inline candi/openapi/node_group.yaml:83]: I don't think it's a good idea - to move the value on the next line

File: candi/openapi/node_group.yaml [hunk 3]
```diff
@@ -114,8 +115,10 @@ spec:
                   description: Number of overprovisioned instances in the group.
                 error:
                   type: string
-                  description: Error message about possible problems with the group
-                    handling.
+                  description: "Error message about possible problems with the group handling."
+                kubernetesVersion:
+                  type: string
+                  description: "The current version of kubernetes on the nodes, or the version to which the nodes will be upgraded."
```
Comment (yalosev) [inline candi/openapi/node_group.yaml:121]: A kinda confusing description... @z9r5 could you validate this?

File: candi/openapi/node_group.yaml (hunk 2 above)
Comment (alex123012) [reply candi/openapi/node_group.yaml:83]: All statuses written earlier were in English. Should I rewrite statuses to the Russian language for the rest?

File: candi/openapi/doc-ru-node_group.yaml (hunk 1 above)
Comment (alex123012) [reply candi/openapi/doc-ru-node_group.yaml:76]: All statuses written earlier were in English. Should I rewrite statuses descriptions to the Russian language for the rest?

Comment (z9r5) [reply candi/openapi/doc-ru-node_group.yaml:76]: @alex123012 Please delete the status section from the doc-ru-*.yaml. The status section of a CRD is not rendered on the side. Thanks

Comment (z9r5) [reply candi/openapi/doc-ru-node_group.yaml:76]: Or simply don't add the kubernetesVersion parameter and live it as is.

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @alex123012. I've started the workflow for label 'e2e/run/openstack'...

**Review by name212 — DISMISSED** [review]

**Review by yalosev — DISMISSED** [review]

File: modules/040-node-manager/hooks/get_crds.go [hunk 4]
```diff
@@ -443,7 +445,7 @@ func getCRDsHandler(input *go_hook.HookInput) error {
 		ngForValues["updateEpoch"] = updateEpoch
 
 		// Reset status error for current NodeGroup.
-		setNodeGroupErrorStatus(input.PatchCollector, nodeGroup.Name, "")
+		setNodeGroupStatus(input.PatchCollector, nodeGroup.Name, "error", "")
```
Comment (RomanenkoDenys) [inline modules/040-node-manager/hooks/get_crds.go:448]: NIT: "error" used in 5 places, i think it should be contstant. and "kubernetes" too.

**Review by RomanenkoDenys — COMMENTED** [review]
some nits

**Review by yalosev — APPROVED** [review]

**Pull request merged**